### DISCUSSION TITLE
Configuration for BR Sega Saturn Adapter and 8bitdo Saturn Modkit

### DISCUSSION THREAD

**Discussion by iamaburrama** [darthcloud/BlueRetro#773] [Q&A] [answered]
Are there any recommended settings or a specific pairing mode the 8bitdo modkit overlay needs to be in to work with the Saturn Bluetooth adapter?

My setup is a Fenrir with a Pseudo Saturn Kai cart. 

I've tried setting the controller into into each of the different pairing modes (D, X, Mac and Switch Input).  The controller will pair in any mode but I can't get the control pad to respond.

If I press start, use any of the shoulder or gamepad buttons these seem to respond and allow me to get into the first game I have in my library.  In game I'm still able to press start and go to any options that only need face buttons.

In the Source and Destinations mappings inside the Advanced Config I've tried changing them both to the Saturn configuration but am still seeing the same issues.  Not sure if maybe this needs to be set as a generic mapping as Source and Saturn as Destination? 

Was wondering if anyone may had any luck pairing and using this controller successfully?

Thanks

**Comment by DoomyDoomer** [comment, marked as answer]
Just to confirm, you're having d-pad issue but start, L, R, and face buttons work? If so, you need to put the d-pad into "d-pad mode". I believe it's start + up or start + left for 5 seconds but you may need to reference 8bitdos manual. No need to adjust the webconfig mapping. You should also put the controller into X Input mode.

**Reply by maniaco007** [reply]
fiz exatamente isto no meu, e funcionou perfeitamente.

**Reply by iamaburrama** [reply]
Ok that was the problem.  I was looking at this doc: https://download.8bitdo.com/Manual/Mod-Kit/Mod-Kit-for-original-sega-saturn-controller.pdf?20220513

Start + Up was what I needed to do.  

Thank you for your help!

**Comment by DoomyDoomer** [comment]
I've also added a new page to my Saturn BT manual to hopefully help people in the future:
![image](https://github.com/darthcloud/BlueRetro/assets/57010233/0d96f81c-fa7d-498a-9ae0-8cefbf78ccf1)
